### DISCUSSION TITLE
[BUG] - Synology Container Manager cannot change port to anything but 80

### DISCUSSION THREAD

**Discussion by drgruney** [mealie-recipes/mealie#2713] [General]
### First Check

- [X] This is not a feature request
- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the Mealie documentation, with the integrated search.
- [X] I already read the docs and didn't find an answer.
- [ ] This issue can be replicated on the demo site (https://demo.mealie.io/)

### What is the issue you are experiencing?

When I install this with the Synology Container Manager I am unable to set any port other than 80 for the hosed Mealie page.

Can the port number be removed from the primary json file hosted at Docker Hub? I have been unable to change this by editing the file myself and trying to repackage to import into Container Manager.

### Steps to Reproduce

Search Docker Hub registry for Mealie

Open/run image

80 is default port and cannot be changed (this is the only docker image I have had this issue with)

### Please provide relevent logs

There are no logs

### Deployment

Docker (Synology)

**Comment by reederda** [comment]
The image hosted on Docker Hub seems to be somewhat out of date. Have you tried using the Github version, which was last updated 3 days ago? 
`docker pull ghcr.io/mealie-recipes/mealie:nightly`

**Comment by drgruney** [comment]
> The image hosted on Docker Hub seems to be somewhat out of date. Have you tried using the Github version, which was last updated 3 days ago? `docker pull ghcr.io/mealie-recipes/mealie:nightly`

I've been unable to get Git Hub containers loaded into the Synology Container Manager. Not sure if it's supported :(

**Comment by boc-the-git** [comment]
Docker Hub is deprecated and will not receive any updates moving forward (re Mealie, not in general). 

I don't know much about Synology, but I'd expect there must be a way to point at GHCR.

**Comment by michael-genson** [comment]
@drgruney if you have the new Container Manager, it supports docker compose, so you can follow the Mealie docs:

1. In Container Manager, go to the "Project" tab and click "Create" ![image](https://github.com/mealie-recipes/mealie/assets/71845777/600aca6d-eea7-4cf7-9901-0bd5e4393f01)
2. Set the Source to "Create docker-compose.yml" and paste Mealie's Docker Compose in there ([SQLite](https://nightly.mealie.io/documentation/getting-started/installation/sqlite/) | [Postgres](https://nightly.mealie.io/documentation/getting-started/installation/postgres/)) 
![image](https://github.com/mealie-recipes/mealie/assets/71845777/130057ac-96eb-4f92-aed3-44ed0cf176d6)

Note that you may need to modify the volume mounting to explicitly mount to a real path, rather than the `driver: local`. You will know if your volume is mounted if you see Mealie data after running the project once in your selected path, along with the generated docker compose file. If you run into issues with that, let me know, it's an easy fix

**Comment by michael-genson** [comment]
For anyone else who may stumble upon this thread: if you're stuck on DSM 7.1 or below, you can still use docker compose, you just need to use SSH instead

**Comment by drgruney** [comment]
Thank you. I will take a look at this tomorrow and see if I can get
anywhere.

I was able to reroute the exposed port with Web Station, but will get this
going so I can stay up to date.

**Comment by michael-genson** [comment]
No problem! If you run into any issues let me know, I run Mealie on Synology using the container manager

**Comment by hay-kot** [comment]
Moving to discussions as I don't think this is a bug.

**Comment by dshelley66** [comment]
I just started with Mealie last week and my first install was on a Synology 1513+ with DSM 7.1. I used portainer to do this and had no issues with ports or ghcr. Basically followed this how-to - https://mariushosting.com/how-to-install-mealie-v1-on-your-synology-nas/

I now have it installed on a 1521+ with DSM 7.2 with the same setup.
